Big Neon Core/API/API.swift

```diff
@@ -24,6 +24,7 @@ public enum APIService {
     public static let updateUser    = stagingBaseURL + "/users/me"
     public static let refreshToken  = stagingBaseURL + "/auth/token/refresh"
     public static let redeem        = stagingBaseURL + "/tickets/"
+    public static let redeemTicket  = stagingBaseURL + "/events/"
```

Comment:
What might be better is using string replacements in each of these urls so that just by looking here you can already see exactly what is required. Something like
```suggestion
    public static let redeemTicket  = stagingBaseURL + "/events/{eventId}/redeem/{ticketId}"
```
Then you do a string replace for eventId and ticketId

Reply:
Cool. Great Idea. I'll into how to do it in swift. That syntax would not work in swift. I'll probably use suggestive commenting.

Reply:
Can you not use something similar to https://github.com/big-neon/bn-ios/blob/70025bf098ca7e0d0ab02f6c7a8209e44d921475/Big%20Neon%20Core/Models/Event/Event.swift#L89